### ISSUE TITLE
All codeium conversations get deleted constantly

### ISSUE THREAD

**Issue by The5-1** [Exafunction/CodeiumVisualStudio#103]
![no conversations](https://github.com/user-attachments/assets/636c4a45-c243-4e53-8b2a-2deacb9bd74b)

Version: 1.91.1 (system setup)
Commit: f1e16e1e6214d7c44d078b1f0607b2388f29d729
Date: 2024-07-09T22:06:49.809Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Windows_NT x64 10.0.19045

Not sure if this is related to the VSCode extension.

Every few messages i submit the whole chat disappears, and all my previous conversations are also gone.
Basically rendering this unusable.

Codeium worked well for an hour or so but then these issues began.

**Comment by QaldeK** [comment]
i have the same probleme...
